tools/msgdis.py

```diff
@@ -391,7 +391,7 @@ def extract_all_text(text_out, staff_text_out):
                 out += "\n#endif"
             out += "\n\n"
 
-        with open(text_out, "w") as outfile:
+        with open(text_out, "w", encoding="utf8") as outfile:
```

Comment:
this addresses:
![image](https://user-images.githubusercontent.com/5306619/235380724-76b3602c-a5c7-4699-b9f7-d49a474d4909.png)

though I thought python defaulted to utf8 regardless of system/environment/... :shrug: